z_infra_provisioning/cloud_infra_center/ocp_upi/tools/modify-dhcp.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# =================================================================
+# Licensed Materials - Property of IBM
+#
+# (c) Copyright IBM Corp. 2021 All Rights Reserved
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# =================================================================
+
+import os
+import re
+"""
+This script will get current cluster nodes IP addresses and Mac from Ports, and then rewrite them into dhcp_template.yaml. 
+So that you can use dhcp_template.yaml to configure DHCP on bastion node.
+"""
+
+template = """
+static_hosts:
+  bootstrap:
+    mac: "00:11:22:33:44:55"
+    ip: "10.1.0.10"
+  master-0:
+    mac: "00:11:22:33:44:56"
+    ip: "10.1.0.11"
+  master-1:
+    mac: "00:11:22:33:44:57"
+    ip: "10.1.0.12"
+  master-2:
+    mac: "00:11:22:33:44:58"
+    ip: "10.1.0.13"
+  worker-0:
+    mac: "00:11:22:33:44:59"
+    ip: "10.1.0.14"
+  worker-1:
+    mac: "00:11:22:33:44:60"
+    ip: "10.1.0.15"
+  worker-2:
+    mac: "00:11:22:33:44:61"
+    ip: "10.1.0.16"
+"""
+template = "static_hosts:"
+cmd = "jq -r .infraID metadata.json"
```

Comment:
Hi @shhull , The template valube is set value twice on line18 and line 42, is it reasonble?

Reply:
Remove the line18, and keep line42.